src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -144,6 +149,12 @@ public function handle()
             $helper->setCloner($cloner);
         }
 
+        $rootPath = $this->applicationRootPath;
+        if (!$rootPath) {
+            $rootPath = dirname(dirname(dirname(dirname(dirname(dirname(__DIR__))))));
```

Comment:
This will work only for installation via Composer to the project's vendor dir? right?